packages/web-components/src/design-tokens.ts

```diff
@@ -15,22 +19,26 @@ export const borderRadiusSmall = create<string>('borderRadiusSmall').withDefault
 /**
  * @public
  */
-export const borderRadiusMedium = create<string>('borderRadiusSmall').withDefault('4px');
+export const borderRadiusMedium = create<string>('borderRadiusMedium').withDefault('4px');
 
 /**
  * @public
  */
-export const borderRadiusLarge = create<string>('borderRadiusSmall').withDefault('6px');
+export const borderRadiusLarge = create<string>('borderRadiusLarge').withDefault('6px');
 
 /**
  * @public
  */
-export const borderRadiusXLarge = create<string>('borderRadiusSmall').withDefault('8px');
+export const borderRadiusXLarge = create<string>('borderRadiusXLarge').withDefault('8px');
 
 /**
  * @public
  */
-export const borderRadiusCircular = create<string>('borderRadiusSmall').withDefault('10000px');
+export const borderRadiusCircular = create<string>('borderRadiusCircular').withDefault('10000px');
```

Comment:
Fix a couple tokens, added (one?) missing, and added some comments for organization.

Reply:
Tokens work was merged in #4